CHANGELOG.md

```diff
@@ -431,6 +446,7 @@ Future versions are likely to introduce backward incompatibilities with < Ruby 1
 [#257]: https://github.com/DataDog/dogstatsd-ruby/issues/257
 [#258]: https://github.com/DataDog/dogstatsd-ruby/issues/258
 [#260]: https://github.com/DataDog/dogstatsd-ruby/issues/260
+[#271]: https://github.com/DataDog/dogstatsd-ruby/issues/271
```

Comment:
```suggestion
[#277]: https://github.com/DataDog/dogstatsd-ruby/issues/277
```

Reply:
#277 points to this PR, and #271 points to the issue where we discussed the feature implementation before. Are you sure you want me to make it point to this PR? Also, should I change the URL from `issues/277` to `pull/277` instead?

Reply:
It was a nit, I don't really mind, let's keep it that way at least it links to the initial conversation first!

Reply:
:sparkles: